api/server/router/network/network_routes.go

```diff
@@ -216,16 +219,22 @@ func (n *networkRouter) postNetworkCreate(ctx context.Context, w http.ResponseWr
 		return nameConflict(create.Name)
 	}
 
+	warnings := make([]string, 0)
+	if errs := validateIPAM(create.IPAM, create.EnableIPv6); errs.ErrorOrNil() != nil {
+		for _, err := range errs.WrappedErrors() {
+			warnings = append(warnings, err.Error())
+		}
+	}
```

Comment:
I think I'd rather not break the "error" abstraction here, and instead make `` return a regular error (`return `). go-multierror already has a default format to combine multiple errors, which would look something like this;

```
    2 errors occurred:
        	* subnet blablabla has some bits set in its host fragment
        	* IPv6 has not been enabled for this network, but an IPv6 subnet has been provided
```

But if we really want to define our own formatting, we can set the `ErrorFormat` instead, and implement our own; https://github.com/moby/moby/blob/97455cc31ffa08078db6591f018256ed59c35bbc/vendor/github.com/hashicorp/go-multierror/multierror.go#L15-L22

The default one looks like this; https://github.com/moby/moby/blob/97455cc31ffa08078db6591f018256ed59c35bbc/vendor/github.com/hashicorp/go-multierror/format.go#L12-L27

And make this code;

```go
	if err := validateIPAM(create.IPAM, create.EnableIPv6); err != nil {
		warnings = append(warnings, err.Error())
	}
```